### PR TITLE
Add improvements in json-ld definition for organization

### PR DESCRIPTION
**Pull request by anita-chouhan**
Reference: [https://github.com/miranj/miranj.in/blob/dev/templates/miranjv2/_schemas/definitions/organization.twig](https://github.com/miranj/miranj.in/blob/dev/templates/miranjv2/_schemas/definitions/organization.twig)

### CODE REVIEW

File: templates/_schemas/definitions/organization.twig [hunk 1]
```diff
@@ -1,12 +1,25 @@
 {% macro organization(props, schema = null) %}
+  {% from '_schemas/definitions/thing' import thing %}
   
   {% set schema = schema ?? craft.schema.Organization %}
   {% if props.url ?? false %}
     {% set schema = schema.id(props.url ~ '#identity') %}
   {% endif %}
+  {% set schema = thing(props, schema) %}
   {% set schema = schema
     .name(props.title)
```
Comment (rungta) [inline templates/_schemas/definitions/organization.twig]: `name` is set by `Thing`, can be removed from here now.

File: templates/_schemas/definitions/organization.twig [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
     .url(props.url|default(null))
```
Comment (rungta) [inline templates/_schemas/definitions/organization.twig]: `url` is set by `Thing`, can be removed from here now.

**Review by rungta — CHANGES_REQUESTED** [review]

File: templates/_schemas/definitions/organization.twig (hunk 1 above)
Comment (anita-chouhan) [reply templates/_schemas/definitions/organization.twig]: Done.

File: templates/_schemas/definitions/organization.twig (hunk 2 above)
Comment (anita-chouhan) [reply templates/_schemas/definitions/organization.twig]: Done.

**Review by rungta — APPROVED** [review]

**Pull request merged**